### ISSUE TITLE
Fio client/server mode support

### ISSUE THREAD

**Issue by fritchie** [kastenhq/kubestr#67]
Does kubestr plan on supporting fio in client/server mode (many server pods running fio and sending result to a single client)?

**Comment by bathina2** [comment]
Yes, there is a plan to support driving i/o from multiple pods. Will update when we address this
